ocp_resources/security_context_constraints.py

```diff
@@ -2,8 +2,174 @@
+        self.allow_privilege_escalation = allow_privilege_escalation
+        self.allow_privileged_container = allow_privileged_container
+        self.allowed_capabilities = allowed_capabilities
+        self.allowed_flex_volumes = allowed_flex_volumes
+        self.allowed_unsafe_sysctls = allowed_unsafe_sysctls
+        self.default_add_capabilities = default_add_capabilities
+        self.default_allow_privilege_escalation = default_allow_privilege_escalation
+        self.forbidden_sysctls = forbidden_sysctls
+        self.fs_group = fs_group
+        self.groups = groups
+        self.priority = priority
+        self.read_only_root_filesystem = read_only_root_filesystem
+        self.required_drop_capabilities = required_drop_capabilities
+        self.run_as_user = run_as_user
+        self.se_linux_context = se_linux_context
+        self.seccomp_profiles = seccomp_profiles
+        self.supplemental_groups = supplemental_groups
+        self.users = users
+        self.volumes = volumes
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file:
+            self.res.update()
+            if self.allow_host_dir_volume_plugin:
+                self.res["allowHostDirVolumePlugin"] = self.allow_host_dir_volume_plugin
+
+            if self.allow_host_ipc:
+                self.res["allowHostIPC"] = self.allow_host_ipc
+
+            if self.allow_host_network:
+                self.res["allowHostNetwork"] = self.allow_host_network
+
+            if self.allow_host_pid:
+                self.res["allowHostPID"] = self.allow_host_pid
+
+            if self.allow_host_ports:
+                self.res["allowHostPorts"] = self.allow_host_ports
+
+            if self.allow_privilege_escalation:
+                self.res["allowPrivilegeEscalation"] = self.allow_privilege_escalation
+
+            if self.allow_privileged_container:
+                self.res["allowPrivilegedContainer"] = self.allow_privileged_container
+
+            if self.allowed_capabilities:
+                self.res["allowedCapabilities"] = self.allowed_capabilities
+
+            if self.allowed_flex_volumes:
+                self.res["allowedFlexVolumes"] = self.allowed_flex_volumes
+
+            if self.allowed_unsafe_sysctls:
+                self.res["allowedUnsafeSysctls"] = self.allowed_unsafe_sysctls
+
+            if self.default_add_capabilities:
+                self.res["defaultAddCapabilities"] = self.default_add_capabilities
+
+            if self.default_allow_privilege_escalation:
+                self.res[
+                    "defaultAllowPrivilegeEscalation"
+                ] = self.default_allow_privilege_escalation
+
+            if self.forbidden_sysctls:
+                self.res["forbiddenSysctls"] = self.forbidden_sysctls
+
+            if self.fs_group:
+                self.res["fsGroup"] = self.fs_group
+
+            if self.groups:
+                self.res["groups"] = self.groups
+
+            if self.priority:
```

Comment:
`priority` default is `0`
is `0` a valid option?if so, the code below will not be reached